.github/workflows/check_pull_request.yml

```diff
@@ -0,0 +1,26 @@
+name: Check Pull Request Quality
```

Comment:
This is what we use in Kids-First for Github workflow. I think it should work on that repository too.